apps/somnia/src/assets/assets.ts

```diff
@@ -0,0 +1,35 @@
+export const rarityColors = [0x818486, 0x5a977a, 0x5987ad, 0x9470bb, 0xc87c24]
```

Comment:
we have these in libs/gi/theme/src/Theme.tsx

If we want to be consistent, we can have a libs/gi/color library or something so frontend and bot can share the same rarity colors

Reply:
i think that would be better, bot cant use tsx